handlers/permissionHandlers.js

```diff
@@ -0,0 +1,28 @@
+const {has} = require('../helpers/has');
+
+const RoleEnum = {
+  Helper: 0, // Subject to change
+  'Super User': 1,
+  Moderator: 2,
+  Admin: 3,
+  'Super Admin': 4,
+};
+
+function hasPermission(msg, command) {
+  const highestRole = msg.member.roles.highest.name;
+
+  if (!has(RoleEnum, highestRole)) {
+    return false;
+  }
+
+  if (!RoleEnum[highestRole] >= RoleEnum[command.minRole]) {
```

Comment:
Would it make more sense for this test to be:

```js
RoleEnum[highestRole] < RoleEnum[command.minRole]
```

Reply:
It sure would! I'll go ahead and make that change